packages/manager/apps/web/client/app/domain/dashboard/domain.routing.js

```diff
@@ -63,6 +63,8 @@ export default /* @ngInject */ ($stateProvider) => {
 
       goToDns: /* @ngInject */ ($state) => () =>
         $state.go('app.domain.product.dns'),
+      goToDnsModify: /* @ngInject */ ($state) => () =>
```

Comment:
Tiny improvment: maybe we can factorize this pattern with commun function that takes param 'domain.product' or 'alldom.domain' in order to decrease duplication in this routing file

Reply:
I can't understand how you want me to improve it

Reply:
it was to avoid code duplication and ease refacto in the future